code/game/objects/items/weapons/storage/secure.dm

```diff
@@ -197,7 +197,7 @@
 	icon_locking = "safeb"
 	icon_sparking = "safespark"
 	force = 8.0
-	w_class = 8.0
+	w_class = 5.0

```

Comment:
I missed this initially (had I seen it, I would have waited).

Why was this changed?